### PR TITLE
User can now change tab layout

### PR DESCRIPTION
**Pull request by CamAnNguyen**
- Add a button next to final tab to open tab layout
- Drag and drop to change tab order 
- Drag tab to "hidden area" to hide tab
- Change will be applied when exiting layout editor
- Tab layout is saved in user table

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -358,6 +371,28 @@
   add_index "samples", ["molecule_id"], name: "index_samples_on_sample_id", using: :btree
   add_index "samples", ["user_id"], name: "index_samples_on_user_id", using: :btree
 
+  create_table "scifinding_credentials", force: :cascade do |t|
```
Comment (Sergeykot) [inline db/schema.rb]: these tables are odd
`rake db:drop && rake db:create && rake db:migrate` and then updating schema.rb will help

Comment (Sergeykot) [inline db/schema.rb]: @CamAnNguyen @camannguyenthanh

Comment (CamAnNguyen) [inline db/schema.rb]: Thank you, just updated the `schema.rb`

**Comment by Sergeykot** [comment]
tested :+1:

**Pull request merged**